tests/visualization_tests/test_slice.py

```diff
@@ -1,99 +1,364 @@
+                is_numerical=False,
+            ),
+        ],
+    )
+
+
+def test_get_slice_plot_info_mixture_category_types() -> None:
+
+    study = _create_study_mixture_category_types()
+    info = _get_slice_plot_info(study, None, None, "Objective Value")
+    assert info == _SlicePlotInfo(
+        target_name="Objective Value",
+        subplots=[
+            _SliceSubplotInfo(
+                param_name="param_a",
+                x=[None, "100"],
+                y=[0.0, 0.5],
+                trial_numbers=[0, 1],
+                is_log=False,
+                is_numerical=False,
+            ),
+            _SliceSubplotInfo(
+                param_name="param_b",
+                x=[101, 102.0],
+                y=[0.0, 0.5],
+                trial_numbers=[0, 1],
+                is_log=False,
+                is_numerical=True,
+            ),
+        ],
+    )
+
+
+@pytest.mark.parametrize("value", [float("inf"), -float("inf")])
+def test_get_slice_plot_info_nonfinite_removed(value: float) -> None:
+
+    study = prepare_study_with_trials(with_c_d=True, value_for_first_trial=value)
+    info = _get_slice_plot_info(
+        study, params=["param_b", "param_d"], target=None, target_name="Objective Value"
+    )
+    assert info == _SlicePlotInfo(
+        target_name="Objective Value",
+        subplots=[
+            _SliceSubplotInfo(
+                param_name="param_b",
+                x=[0.0, 1.0],
+                y=[2.0, 1.0],
+                trial_numbers=[1, 2],
+                is_log=False,
+                is_numerical=True,
+            ),
+            _SliceSubplotInfo(
+                param_name="param_d",
+                x=[4.0, 2.0],
+                y=[2.0, 1.0],
+                trial_numbers=[1, 2],
+                is_log=False,
+                is_numerical=True,
+            ),
+        ],
+    )
+
+
+@pytest.mark.parametrize("objective", (0, 1))
+@pytest.mark.parametrize("value", (float("inf"), -float("inf")))
+def test_get_slice_plot_info_nonfinite_multiobjective(objective: int, value: float) -> None:
+
+    study = prepare_study_with_trials(with_c_d=True, n_objectives=2, value_for_first_trial=value)
+    info = _get_slice_plot_info(
+        study,
+        params=["param_b", "param_d"],
+        target=lambda t: t.values[objective],
+        target_name="Objective Value",
+    )
+    assert info == _SlicePlotInfo(
+        target_name="Objective Value",
+        subplots=[
+            _SliceSubplotInfo(
+                param_name="param_b",
+                x=[0.0, 1.0],
+                y=[2.0, 1.0],
+                trial_numbers=[1, 2],
+                is_log=False,
+                is_numerical=True,
+            ),
+            _SliceSubplotInfo(
+                param_name="param_d",
+                x=[4.0, 2.0],
+                y=[2.0, 1.0],
+                trial_numbers=[1, 2],
+                is_log=False,
+                is_numerical=True,
+            ),
+        ],
+    )
```

Comment:
I don't think we need this test, since this seems not to be increasing the coverage. 
```suggestion
```

Reply:
The other test files for the plot methods have the same test. Should we remove all of them?

Reply:
I think this test is necessary because it is not for coverage, but to verify behavior.

Reply:
I think we have already tested the behavior of `inf` values in the single-objective case, and we use the same code for multi-objective case, so we don't need to check the multi-objective case again.

Reply:
@HideakiImamura What do you think?

Reply:
Thanks for the comment, and sorry for the late reply. We have this type of tests in other visualization files, so I think we should remove it together if we decide to remove them. So, I think it could be outside the scope of this PR.